.npmrc

```diff
@@ -1,4 +1,4 @@
 scripts-prepend-node-path=true
 //npm.pkg.github.com/:_authToken=${NPM_TOKEN}
-@provenanceio-bot:registry=https://npm.pkg.github.com
+@provenance-io:registry=https://npm.pkg.github.com
```

Comment:
should be `provenanceio`

Reply:
see https://www.npmjs.com/search?q=%40provenanceio

Reply:
Thanks! Fixed.